virtualizarr/tests/test_writers/test_icechunk.py

```diff
@@ -282,9 +290,25 @@ def test_write_loadable_variable(
     assert isinstance(pres_array, Array)
     assert pres_array.shape == (3, 4)
     assert pres_array.dtype == np.dtype("int32")
-    expected_ds = open_dataset(simple_netcdf4)
+    expected_ds = open_dataset(netcdf_path)
     expected_array = expected_ds["foo"].to_numpy()
     npt.assert_equal(pres_array, expected_array)
+    expected_ds.close()
+
+    # Now we can overwrite the simple_netcdf4 file with new data to make sure that
+    # the checksum_date is being used to determine if the data is valid
```

Comment:
This part of the test seems to be testing a separate feature - perhaps it should be split out into a separate test?

Reply:
We could also just add the whole `last_updated_at` feature in a follow-up PR.

Reply:
This is mostly me being lazy and not wanting to have to duplicate stuff that is already in a test. I can split it outto its own test if you want

Reply:
Okay that's fine. At least add more to the whatsnew noting this new feature though.